patches/node/refactor_allow_embedder_overriding_of_internal_fs_calls.patch

```diff
@@ -60,8 +60,8 @@ index 4a2b0e6ddb3ed8881f896c6f556c9debaf778b81..209df9033a26c29c6e60bd173f710558
      return cache.get(jsonPath);
    }
  
--  const [string, containsKeys] = internalModuleReadJSON(
-+  const [string, containsKeys] = internalFsBinding.internalModuleReadJSON(
+-  const { 0: string, 1: containsKeys } = internalModuleReadJSON(
++  const { 0: string, 1: containsKeys } = internalFsBinding.internalModuleReadJSON(
```

Comment:
‼️,  why is this weird destructuring needed...?

Reply:
@nornagon this is not our change - it's an update to adapt to [Node.js own code](https://github.com/nodejs/node/blob/5f2bb885f8027e2789a1fa8679d242162b4a8633/lib/internal/modules/package_json_reader.js#L21-L23)